src/analysis/normed_space/lp_space.lean

```diff
@@ -163,6 +164,7 @@ end
 @[simp] lemma neg_iff {f : Π i, E i} : mem_ℓp (-f) p ↔ mem_ℓp f p :=
 ⟨λ h, neg_neg f ▸ h.neg, mem_ℓp.neg⟩
 
+
```

Comment:
Accidental change?

Reply:
Yes. Let me see if I can roll back to an earlier version, and insert the actual change with minimal modification...